src/definitions/collections/form.less

```diff
@@ -412,6 +412,39 @@
   background: @inputFocusBackground;
   box-shadow: @inputFocusBoxShadow;
 }
+& when (@variationInputAction) {
+  .ui.form .ui.action.input:not(.left) input:not([type]):focus,
+  .ui.form .ui.action.input:not(.left) input[type="date"]:focus,
+  .ui.form .ui.action.input:not(.left) input[type="datetime-local"]:focus,
+  .ui.form .ui.action.input:not(.left) input[type="email"]:focus,
+  .ui.form .ui.action.input:not(.left) input[type="number"]:focus,
+  .ui.form .ui.action.input:not(.left) input[type="password"]:focus,
+  .ui.form .ui.action.input:not(.left) input[type="search"]:focus,
+  .ui.form .ui.action.input:not(.left) input[type="tel"]:focus,
+  .ui.form .ui.action.input:not(.left) input[type="time"]:focus,
+  .ui.form .ui.action.input:not(.left) input[type="text"]:focus,
+  .ui.form .ui.action.input:not(.left) input[type="file"]:focus,
+  .ui.form .ui.action.input:not(.left) input[type="url"]:focus {
+    border-top-right-radius: 0;
+    border-bottom-right-radius: 0;
+  }
+
+  .ui.form .ui.action.input.left input:not([type]),
+  .ui.form .ui.action.input.left input[type="date"],
+  .ui.form .ui.action.input.left input[type="datetime-local"],
+  .ui.form .ui.action.input.left input[type="email"],
+  .ui.form .ui.action.input.left input[type="number"],
+  .ui.form .ui.action.input.left input[type="password"],
+  .ui.form .ui.action.input.left input[type="search"],
+  .ui.form .ui.action.input.left input[type="tel"],
+  .ui.form .ui.action.input.left input[type="time"],
+  .ui.form .ui.action.input.left input[type="text"],
+  .ui.form .ui.action.input.left input[type="file"],
+  .ui.form .ui.action.input.left input[type="url"] {
```

Comment:
Why those lines does not have the `:focus` attribute like above ?

Reply:
Because `.left.action.input` have left radius by default (not focused).
![image](https://user-images.githubusercontent.com/127635/70328891-76b87480-187d-11ea-8724-11f464aebab8.png)

Reply:
Ok, thank for the explanation @exoego !